.travis.yml

```diff
@@ -1,4 +1,16 @@
 sudo: false
 
-language: node_js
-node_js: stable
+language: java
```

Comment:
I see. 
The problem with this YAML is that it breaks the possibility to test multiple node version.
Can you try if it works with this version : 
```yml
sudo: false

language: java

jdk:
  - oraclejdk8

node_js:
  - stable
  - 5
  - 4
  - 0.12

before_install:
  - nvm install $TRAVIS_NODE_VERSION

install: "npm install"

script: "npm test"
```